fec.config.js

```diff
@@ -20,6 +20,6 @@ module.exports = {
       './IamUserAccess': path.resolve(__dirname, './src/entries/IamUserAccess.js'),
     },
     exclude: ['react-router-dom'],
-    shared: [{ 'react-router-dom': { singleton: true } }],
+    shared: [{ 'react-router-dom': { singleton: true } }, { '@unleash/proxy-client-react': { singleton: true, requiredVersion: '*' } }],
```

Comment:
One more thing. The unleash should already be configured in the default webpack config. Can you verify it works without this configuration?

Reply:
@Hyperkid123 it works, thank you!